app/models/stock.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Stock < ApplicationRecord
+  has_many :user_stocks
+  has_many :users, through: :user_stocks
+
+  validates :name, :ticker, presence: true
+
+  def self.new_lookup(ticker_symbol)
+    client = IEX::Api::Client.new(publishable_token: Rails.application.credentials.iex_client[:sandbox_api_key],
```

Comment:
We should really be storing these constants in a configuration area, such as `config/initializers/iex.rb`